cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -38,26 +38,12 @@ extern "C" {
 static FliProcessCbHdl *sim_init_cb;
 static FliProcessCbHdl *sim_finish_cb;
 static FliImpl         *fli_table;
-}
-
-void FliImpl::sim_end(void)
-{
-    if (GPI_DELETE != sim_finish_cb->get_call_state()) {
-        sim_finish_cb->set_call_state(GPI_DELETE);
-        if (mti_NowUpper() == 0 && mti_Now() == 0 && mti_Delta() == 0) {
-            mti_Quit();
-        } else {
-            mti_Break();
-        }
-    }
-}
```

Comment:
In the interest of keeping this diff small, I'd err on the side of not moving this function

Reply:
Unless the functions are moving back to be part of the FliImpl, I disagree.  I prefer to keep class implementation code together.